### ISSUE TITLE
[Help]: Clarification on VALL-E and VALL-E V2 in Amphion Repository

### ISSUE THREAD

**Issue by nngocson2002** [open-mmlab/Amphion#370]
Upon reviewing the repository, I noticed it contains two versions: **VALL-E** and **VALL-E V2**. Could you kindly clarify if both implementations correspond to the same paper ([paper link](https://arxiv.org/abs/2301.02111))? Or does **VALL-E** correspond to this paper ([paper link](https://arxiv.org/abs/2301.02111)) and **VALL-E V2** to this one ([paper link](https://arxiv.org/abs/2406.05370))?

Additionally, I am attempting to train **VALL-E** from scratch using the `egs/tts/VALLE` directory. Specifically, during the AR training step, I noticed that the `train_stage` and `ar_model_ckpt_dir` arguments mentioned in the `egs/tts/VALLE/run.sh` file are not reflected in the `bins/tts/train.py` script. Could you provide guidance on resolving this mismatch?

**Comment by lmxue** [comment]
Hi @nngocson2002 , thanks for your email and for bringing up this issue. 
As mentioned in README of [VALL-E](https://github.com/open-mmlab/Amphion/tree/main/egs/tts/VALLE#vall-e-recipe)  and [VALL-E v2](https://github.com/open-mmlab/Amphion/tree/main/egs/tts/VALLE_V2#change-notes),  [VALL-E] is the implementation of the [paper](https://arxiv.org/abs/2301.02111), while, VALLE_V2 includes improvements based on vanilla VALL-E.
Additionally, `train_stage ` and are `ar_model_ckpt_dir ` are defined in [valle_train.py](https://github.com/open-mmlab/Amphion/blob/main/models/tts/valle/valle_trainer.py#L355) file.

**Comment by nngocson2002** [comment]
So, if I want to reproduce VALL-E, I just need to choose either VALL-E or VALL-E V2, right? I initially thought these repositories were based on different papers, so I assumed I would need to reproduce both.

**Comment by jiaqili3** [comment]
> So, if I want to reproduce VALL-E, I just need to choose either  or VALL-E V2, right? I initially thought these repositories were based on different papers, so I assumed I would need to reproduce both.

Yes, and if you want to reproduce VALL-E I suggest using VALL-E (v1) which uses the vanilla implementation.

**Comment by nngocson2002** [comment]
Thank you for your clarification!

One point I’d like to discuss: as I mentioned, when I attempt to run the training from scratch (as shown in the image below), it executes the `bins/tts/train.py` script. This results in the `train_stage` and `ar_model_ckpt_dir` arguments being unrecognized.  

Additionally, could you provide the structure of the JSON file referenced in `egs/tts/VALLE/exp_config.json`? I noticed that the preprocessing script reads this JSON file, but I couldn't determine its structure.  

I would greatly appreciate your guidance.  

![Image](https://github.com/user-attachments/assets/92fe3beb-fce5-4d55-935d-9692f647eb1d)

**Comment by VJJJJJJ1** [comment]
> Additionally, could you provide the structure of the JSON file referenced in `egs/tts/VALLE/exp_config.json`? I noticed that the preprocessing script reads this JSON file, but I couldn't determine its structure.

same problem, how do you solve it?
